include/benchmark/benchmark.h

```diff
@@ -1181,6 +1201,39 @@ class CSVReporter : public BenchmarkReporter {
   std::set< std::string > user_counter_names_;
 };
 
+class HTMLReporter : public BenchmarkReporter {
```

Comment:
Thanks for the feedback. Are you saying that such a service should be outside of the project?